xml/common_license_gfdl1.2.xml

```diff
@@ -15,7 +15,7 @@
   xmlns="http://docbook.org/ns/docbook"
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink">
- <title>GNU Free Documentation License</title>
+ <title>GNU free documentation license</title>
```

Comment:
As this one is the basis of an acronym, I think, I would be tempted to leave it. The GPL is very well-known; the GFDL not so much, but still common.

Reply:
```suggestion
 <title>GNU Free Documentation License</title> 
```